README.md

```diff
@@ -42,6 +49,15 @@ Please make sure to `export PYTHONHASHSEED=0` when running the code. You can add
 * Monitor with `squeue -u ronuchit`, or cancel jobs with `scancel -u ronuchit` (standard Slurm commands).
 * When all jobs are done, run `python analysis/analyze_supercloud_experiments.py` (still on supercloud) to print out the results table.
 
+### Running Experiments on BEHAVIOR
+* Currently, only the `oracle` approach is implemented to integrate with BEHAVIOR
+* Note that you'll probably want to provide the `--timeout 1000`, `--max_num_steps_check_policy 1000` to prevent early stopping.
+* Set `--option_model_name behavior_oracle` to use the behavior option model and speed up planning by a significant factor
+* Set `--behavior_task_name` to the name of the particular bddl task you'd like to run (e.g. `re-shelving_library_books`)
+* Set `--behavior_scene_name` to the name of the house setting (e.g. `Pomaria_1_in`, or `Rs_int`) you want to try running the particular task in.
```

Comment:
have we tested Rs_int btw?

Reply:
yes, but not with `re-shelving_library_books` (some tasks exist only in some houses). I'll mention something to this effect